pcsx2/gui/FrameForGS.cpp

```diff
@@ -591,11 +592,11 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 
 	const u64& smode2 = *(u64*)PS2GS_BASE(GS_SMODE2);
 
-	SetTitle( pxsFmt( L"%s | %ls (%ls) | Limiter: %ls | fps: %6.02f%ls | State %d",
+	SetTitle( pxsFmt( L"%s | %ls (%ls) | Limiter: %ls | Game Speed: %6.02f%% %ls | State %d",
```

Comment:
I'd use `Speed` instead of `Game Speed`. The title gets long enough even when if's only `fps`.